Source/engine/assets.cpp

```diff
@@ -31,6 +31,11 @@ bool OpenMpqFile(const char *filename, MpqArchive **archive, uint32_t *fileNumbe
 
 } // namespace
 
+SDL_RWops* OpenAssetFromDisk(const char* filename)

```

Comment:
Fyi the game will already look on disk before looking in the MPQs

It looks relatively to the mpq files, and also in ./assets next to the exe.

Reply:
Thanks, I did try to load from disk before with an absolute path, but it didn't work too well on Windows. As you and @glebm mentioned, I'm going to add it as an "asset" and add to CMake as well.

Reply:
I added the file as an asset now, should illustrate the crash a bit better! It eventually crashes in the load little endian functions, probably as a result of me setting up the animation completely incorrect... I'll be doing some more digging into how the cel/cl2 files are used and hopefully get something working haha